### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by CoastalPlainSoils** [IMCR-Hackathon/Hackathon-Central-2019#10]
Problematic dataset due to column names not displayed properly. 
<img width="1024" alt="Screen Shot 2019-06-11 at 2 20 43 PM" src="https://user-images.githubusercontent.com/42621728/59303775-2789e200-8c54-11e9-8dcd-ee402823c6ce.png">

https://portal.edirepository.org/nis/mapbrowse?packageid=knb-lter-bes.950.420

**Comment by wetlandscapes** [comment]
A typical data set: https://search.dataone.org/view/https://pasta.lternet.edu/package/metadata/eml/edi/216/1.
Specifically: [hydrology_data.csv](https://pasta.lternet.edu/package/data/eml/edi/216/1/4a0572a7135fb1ee02240c1f0b015488)

**Comment by ktoddbrown** [comment]
A Pangaea data set with extra header information but otherwise well formatted  https://doi.pangaea.de/10.1594/PANGAEA.890471

**Comment by jhp7e** [comment]
Here is a typical dataset:
https://pasta.lternet.edu/package/metadata/eml/knb-lter-vcr/153/24  - raw metadata
https://doi.org/10.6073/pasta/7e48a6e1fb576a5be7b20ffbbaa10503 - landing page

And here is a problematic dataset - too many columns, lots of missing data, some range and unit issues. 
https://pasta.lternet.edu/package/metadata/eml/knb-lter-vcr/247/10 - raw metadata
https://doi.org/10.6073/pasta/b650b236f092e0fdee0d5d8ccf521cb3 - landing page

**Comment by CoastalPlainSoils** [comment]
Dataset with code generation line (and R code....) https://portal.edirepository.org/nis/codeGeneration?packageId=knb-lter-bes.549.160&statisticalFileType=r

**Comment by sheilasaia** [comment]
Non-problematic LTER daily stream water chemistry dataset: https://cn.dataone.org/cn/v2/resolve/https%3A%2F%2Fpasta.lternet.edu%2Fpackage%2Fdata%2Feml%2Fknb-lter-knz%2F125%2F1%2Ff9f485f53fcbad9222e9f53394f9a826 (download link for PBG111.csv file). You can view the complete dataone entry at: https://search.dataone.org/view/https://pasta.lternet.edu/package/metadata/eml/knb-lter-knz/125/1.

Sample R code to download: `metajam::download_d1_data(data_url = "https://cn.dataone.org/cn/v2/resolve/https%3A%2F%2Fpasta.lternet.edu%2Fpackage%2Fdata%2Feml%2Fknb-lter-knz%2F125%2F1%2Ff9f485f53fcbad9222e9f53394f9a826", path = "<your destination path>")`

**Comment by ktoddbrown** [comment]
Very clean soils data:
https://esajournals.onlinelibrary.wiley.com/doi/full/10.1002/ecy.2159

**Comment by CoastalPlainSoils** [comment]
Clean data table with code generation also.... https://portal.edirepository.org/nis/mapbrowse?packageid=knb-lter-bes.3130.100

**Comment by vanderbi** [comment]
Typical dataset :  https://search.dataone.org/view/https://pasta.lternet.edu/package/metadata/eml/knb-lter-sev/289/239911  (Sevilleta LTER NPP Quadrat Data)

**Comment by alesiahallmark** [comment]
Flux tower data  (fun because of the high temporal frequency): https://cn.dataone.org/cn/v2/resolve/https%3A%2F%2Fpasta.lternet.edu%2Fpackage%2Fdata%2Feml%2Fknb-lter-hbr%2F241%2F2%2Fb05f44eacc901906ac386b1bbb0414a4

Line-intercept data (simple but strange spatial data): https://cn.dataone.org/cn/v2/resolve/https%3A%2F%2Fpasta.lternet.edu%2Fpackage%2Fdata%2Feml%2Fknb-lter-sev%2F4%2F167418%2Fb702fe7865ddefe3d28061131a50434a

Spatially distributed data, with two data tables, one for data, one for sites: https://search.dataone.org/view/https://pasta.lternet.edu/package/metadata/eml/knb-lter-cap/41/14

**Comment by atn38** [comment]
I advocate for datasets with explicit spatial and temporal columns :)

Subsistence and personal use harvest of salmon in Alaska, 1960-2012
https://search.dataone.org/view/doi:10.5063/F18P5XTN

**Comment by lkuiucsb** [comment]
Three data tables, long time series, multiple measurements
https://portal.edirepository.org/nis/mapbrowse?scope=knb-lter-sbc&identifier=45

**Comment by ktoddbrown** [comment]
This is now here 121a7311a7c971e4862cae1b1f1872e2240e3cf5
> And here is a problematic dataset - too many columns, lots of missing data, some range and unit issues.
> https://pasta.lternet.edu/package/metadata/eml/knb-lter-vcr/247/10 - raw metadata
> https://doi.org/10.6073/pasta/b650b236f092e0fdee0d5d8ccf521cb3 - landing page

**Comment by ktoddbrown** [comment]
This is now here: R/testData_SavilletaNPP.R @vanderbi 

> Typical dataset : https://search.dataone.org/view/https://pasta.lternet.edu/package/metadata/eml/knb-lter-sev/289/239911 (Sevilleta LTER NPP Quadrat Data)

**Comment by clnsmth** [comment]
[Here's the example dataset](https://arcticdata.io/catalog/view/doi:10.18739/A2DP3X) for `read_data_archived()`. An Arctic Data Center data package
